lib/src/testing_library/dom/screen.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:html' show document;
+
+import 'package:over_react_test/src/testing_library/dom/queries.dart' show DomQueries;
+
+/// Queries scoped to the entire `document.body`.
+class _ScreenQueries extends DomQueries {
+  _ScreenQueries() : super(() => document.body);
+}
+
+/// Exposes all the "top-level" queries exposed by the dom-testing-library,
+/// but the scope is defaulted to `document.body`.
+///
+/// > Related: `within()`
+///
+/// > See: <https://testing-library.com/docs/queries/about/#screen>
+final screen = _ScreenQueries();
```

Comment:
This is how I went about mimicking how the JS library "binds" all the "raw" queries to `document.body`, aliased as `screen`.

Reply:
This would allow, for instance - a consumer to render a component that conditionally renders an overlay in a standalone React tree, and easily query for it...

```dart
import 'package:over_react/over_react.dart';
import 'package:over_react_test/react_testing_library.dart' as rtl;
import 'overlay_trigger.dart';

test('my component renders an overlay when clicked', () async {
  rtl.render(
    (OverlayTrigger()
      ..overlay = (Dom.div()..role = 'dialog')('I am the overlay!')
    )(Dom.button()('Click me'))
  );
  
  queryByText('Click me').click();
  // This async query will even wait for the overlay to become visible automatically!
  await screen.findByRole('dialog');
});
```